packages/babel-preset-sui/lib/index.js

```diff
@@ -3,14 +3,25 @@ const cleanList = require('./clean-list')
 
 module.exports = {
   'presets': cleanList([
-    require('babel-preset-es2015'),
+    ["env", {
+      "debug": false,
+      "targets": {
+        "node": "6.0.0",
+        "browsers": [
+          "> 1%",
+          "last 4 versions",
+          "Firefox ESR",
+          "Safari >= 6",
+          "iOS >= 7",
+          "not ie < 11"
+        ]
+      }
+    }],
     isInstalled(['preact', 'react'], 'babel-preset-react'),
     isInstalled('flow-bin', 'babel-preset-flow')
   ]),
   'plugins': [
-    require('babel-plugin-syntax-trailing-function-commas'),
     require('babel-plugin-transform-async-generator-functions'),
-    require('babel-plugin-transform-async-to-generator'),
     require('babel-plugin-transform-decorators-legacy').default,
```

Comment:
We still have a lot of plugins because babel-preset-env@1 is es2015 based.
babel-preset-env@2 is not released yet.